src/Components/v2/Carousel.tsx

```diff
@@ -210,30 +215,39 @@ export class BaseCarousel extends React.Component<
   handleSlideChange = index => {
     this.setState({
       currentSlideIndex: index,
-      lastItemVisible: this.checkLastItemVisible(),
     })
   }
 
   checkLastItemVisible = () => {
-    const lastItemVisible = this.flickity.selectedElements.includes(
-      this.flickity.getLastCell().element
-    )
-    return lastItemVisible
+    if (this.flickity && this.flickity.selectedElements) {
+      const lastItemVisible = this.flickity.selectedElements.includes(
+        this.flickity.getLastCell().element
+      )
+      return lastItemVisible
+    } else {
+      return true
+    }
   }
 
   renderLeftArrow = () => {
-    const { onArrowClick, renderLeftArrow, showArrows } = this.props
+    const { onArrowClick, renderLeftArrow, showArrows, height } = this.props
 
     if (!showArrows) {
       return null
     }
 
+    const leftPosition =
+      this.props.arrowPosition !== null &&
+      this.props.arrowPosition !== undefined
+        ? this.props.arrowPosition
+        : -space(4)
```

Comment:
Why does this default to `-40`? I saw it mentioned in the comment earlier, but it I don't get a sense of why... is that to pull it off the screen?

Reply:
<img width="1204" alt="Screen Shot 2019-07-02 at 1 57 19 PM" src="https://user-images.githubusercontent.com/21182806/60535138-6bab5800-9cd1-11e9-9826-6060e9a0094a.png">

Yea the default is for the arrow to be 40px outside of the container and in space outside of the normal page container.